docker/Dockerfile.build

```diff
@@ -0,0 +1,19 @@
+ARG IMAGE
+FROM ${IMAGE}
```

Comment:
This is important. This allows us to have a __SINGLE__ dockerfile for all distros. And then we pass in `--build-arg=${DISTRO}` in our build process. The specifics of what to install in this build container is nicely abstracted into `docker/setup` script which we copy into the build context below at line 5 and then run at line 11